tests/components/camera/test_init.py

```diff
@@ -169,34 +173,38 @@ def test_snapshot_service(hass, mock_camera):
     await async_setup_component(hass, "camera", {})
 
     client = await hass_ws_client(hass)
-    await client.send_json(
-        {"id": 5, "type": "camera_thumbnail", "entity_id": "camera.demo_camera"}
-    )
+    await client.send_json({
+        "id": 5,
+        "type": "camera_thumbnail",
+        "entity_id": "camera.demo_camera"
+    })
 
     msg = await client.receive_json()
 
     assert msg["id"] == 5
     assert msg["type"] == TYPE_RESULT
     assert msg["success"]
     assert msg["result"]["content_type"] == "image/jpeg"
-    assert msg["result"]["content"] == base64.b64encode(b"Test").decode("utf-8")
+    assert msg["result"]["content"] == base64.b64encode(b"Test").decode(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)